src/pages/postgraphile/usage-library.md

```diff
@@ -218,6 +218,29 @@ if (middleware.options.watchPg) {
 }
 ```
 
+For Nest, this might look something like:
+
+```js
+import { Controller, Get, Post, Req, Next, Res } from '@nestjs/common'
+import { Request, Response } from 'express'
+import { PostGraphileResponseNode } from 'postgraphile'
+import { middleware } from './postgraphile.middleware'
+import { PostGraphileResponseNest } from './postgraphileResponseNest'
```

Comment:
Where is PostGraphileResponseNest defined?

Reply:
whoops that's not needed, will remove